docs/faq.rst

```diff
@@ -45,6 +45,18 @@ repository is only modified from one place. Also keep in mind that
 Borg will keep an exclusive lock on the repository while creating
 or deleting archives, which may make *simultaneous* backups fail.
 
+Can I back up to multiple, swapped backup targets?
+--------------------------------------------------
+
+To protect my backups against data loss, I want to back up to weekly swapped-out
+USB disks and place the media currently not in use offline and off-site to keep
+them safe. How can I apply this or other more elaborate
+`backup schemes <https://en.wikipedia.org/wiki/Backup_rotation_scheme>`_ with
+BorgBackup?
```

Comment:
this is still the "i want ..." point of view.

but it should be rather an answer to the question in the headline.

so just rephrase to "it is possible to ..." and merge with next paragraph.